### PR TITLE
[dwdunwetter] Update docs to clarify CellID selection

### PR DESCRIPTION
**Pull request by florian-h05**
# [dwdunwetter] Update docs to clarify CellID selection

# Description

This PR updates the docs to be more clear about which CellID to choose for the binding to work correctly.
It also updates the German README and the description in the Thing configuration.

Those changes were made out of the motivation that I had problems with the binding because of I used a CellID that was not supported, the docs were not clear enough about that point.

### CODE REVIEW

File: bundles/org.openhab.binding.dwdunwetter/src/main/resources/OH-INF/i18n/dwdunwetter_de.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ thing-type.dwdunwetter.dwdwarnings.description = DWD Unwetterwarnungen für ein
 # thing types config
 
 thing-type.config.dwdunwetter.dwdwarnings.cellId.label = Cell-ID
-thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
+thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv, nur IDs die mit einer 8 (Ausnahme für Berlin: 7) beginnen werden unterstützt. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
```
Comment (lolodomo) [inline bundles/org.openhab.binding.dwdunwetter/src/main/resources/OH-INF/i18n/dwdunwetter_de.properties:14]: Can you please remove this change ?
After the PR is merged, you can then update it in Crowdin.

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
@limdul79 : is it ok for you ?

**Comment by florian-h05** [comment]
@lolodomo 
All your requested changes are addressed now.

**Review by limdul79 — APPROVED** [review]
It's ok for me

**Pull request merged**